src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/ConditionalWeakTable.cs

```diff
@@ -11,8 +12,8 @@
 namespace System.Runtime.CompilerServices
 {
     public sealed class ConditionalWeakTable<TKey, TValue> : IEnumerable<KeyValuePair<TKey, TValue>>
-        where TKey : class
-        where TValue : class
+        where TKey : class?
+        where TValue : class?
```

Comment:
I guess you didn't make _container nullable to keep it simple since we never expose it?

Reply:
> I guess you didn't make _container nullable to keep it simple since we never expose it?

It's only nulled out in the finalizer (not even Dispose).